src/go/rpk/pkg/cli/registry/compatibility_level.go

```diff
@@ -0,0 +1,154 @@
+// Copyright 2023 Redpanda Data, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+package registry
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/config"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/out"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/schemaregistry"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"github.com/twmb/franz-go/pkg/sr"
+)
+
+func compatibilityLevelCommand(fs afero.Fs, p *config.Params) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "compatibility-level",
+		Args:  cobra.ExactArgs(0),
+		Short: "Manage global or per-subject compatibility levels",
+	}
+	cmd.AddCommand(
+		compatGetCommand(fs, p),
+		compatSetCommand(fs, p),
+	)
+	p.InstallFormatFlag(cmd)
+	return cmd
+}
+
+type compatibilityLevelResponse struct {
+	Subject string `json:"subject" yaml:"subject"`
+	Level   string `json:"level,omitempty" yaml:"level,omitempty"`
+	Err     string `json:"error,omitempty" yaml:"error,omitempty"`
+}
+
+func compatGetCommand(fs afero.Fs, p *config.Params) *cobra.Command {
+	var global bool
+	cmd := &cobra.Command{
+		Use:   "get [SUBJECT...]",
+		Short: "Get the global or per-subject compatibility levels",
+		Long: `Get the global or per-subject compatibility levels.
+
+Running this command with no subject returns the global level, alternatively
+you can use the --global flag to get the global level at the same time as
+per-subject levels.
+`,
+		Run: func(_ *cobra.Command, subjects []string) {
+			f := p.Formatter
+			if h, ok := f.Help([]compatibilityLevelResponse{}); ok {
+				out.Exit(h)
+			}
+			p, err := p.LoadVirtualProfile(fs)
+			out.MaybeDie(err, "unable to load config: %v", err)
+
+			cl, err := schemaregistry.NewSRClient(fs, p)
+			out.MaybeDie(err, "unable to initialize schema registry client: %v", err)
+
+			if len(subjects) > 0 && global {
+				subjects = append(subjects, sr.GlobalSubject)
+			}
+			results := cl.CompatibilityLevel(context.Background(), subjects...)
+
+			err = printCompatibilityResult(results, f)
+			out.MaybeDieErr(err)
```

Comment:
```suggestion
			out.MaybeDieErr(printCompatibilityResult(results, f))
```

Any reason to reassign err here rather than daisy chain?

Reply:
Not really, just to keep things clean and separated in the `Run` function of every command. A force of habit I'll say :sweat_smile:

Reply:
fwiw I like the pattern aspect of these last lines

Reply:
Alrighty then 😂